### ISSUE TITLE
Add navigation keystrokes/menu items for headings and other structural elements

### ISSUE THREAD

**Issue by jaybird110127** [trypsynth/paperback#3]
I only accidentally figured out that the left and right bracket keys go between sections in an Epub. I guess with most punctuations turned off in NVDA, the menu doesn't announce the global keyboard shortcuts for these options. In my opinion, they should be h for next section (heading) and shift+H for previous section. Also I'm not sure how Epub works, but if it's like Daisy where there are six possible levels of headings, 1 should go to the next level 1 heading, 2 the next level 2, etc. Shift plus these numbers should go to the prior heading of that level.

**Comment by trypsynth** [comment]
Hi Jayson,
Epub sections are actually an entirely different element from what you're thinking about, which are HTML headings.
To be clear, I plan to support HTML headings as well, so you'll have h and shift+h to go through any headings, and 1-6 and shift 1-6 to go between headings at that particular level.
I'm currently trying to get the project to build with Lexbor, a proper HTML5 parser rather than just wrapping Poco's XML parser, but it's proving to be a bit of a pain. Once I get it though I'll certainly add all of these :)

**Comment by trypsynth** [comment]
Hi,
Thanks for your interest! I agree that it's slightly confusing, I also have my symbols level set to some and so I don't hear them, but I'm relying on the wxWidgets feature of putting tab and a shortcut after a menu item name binding to that shortcut automatically without needing an accel table, but it is confusing. I think the best way to solve this is with good documentation, which will come in time :)
I didn't know discussions aren't enabled, I'll go fix that, that wasn't intended.
